packages/webview_flutter/webview_flutter_platform_interface/lib/src/types/webview_cookie.dart

```diff
@@ -0,0 +1,49 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+/// Configuration to use when creating a new [WebViewPlatformController].
+///
+/// The `autoMediaPlaybackPolicy` parameter must not be null.
+class WebViewCookie {
+  /// Construct a new [WebViewCookie].
+  const WebViewCookie(
+      {required this.name,
+      required this.value,
+      required this.domain,
+      this.path = '/'});
+
+  /// The name of the cookie.
+  ///
+  /// Its value should match "cookie-name" in RFC6265bis:
+  /// https://datatracker.ietf.org/doc/html/draft-ietf-httpbis-rfc6265bis-02#section-4.1.1
+  final String name;
+
+  /// The value of the cookie.
```

Comment:
The first paragraph of the dart docs for the `value`, `domain` and `path` fields are exactly the same (namely `The value of the cookie.`) which is a bit confusing. It would be better to describe this a bit more distinctive. A minimal example would be: 

```
/// The cookie-value of the cookie.

/// The domain-value of the cookie.

/// The path-value of the cookie.
```